lms/templates/financial-assistance/financial_assessment_form.underscore

```diff
@@ -6,42 +6,81 @@
   <%  }); %>
 </div>
 
-<form class="financial-assistance-form" method="POST">
-  <div class="js-form-feedback" aria-live="assertive" tabindex="-1">
+<% if (applications_status_response) { %>
+  <div class="financial-assistance-applications-list">
```

Comment:
Add a header/text here that shows the following table is submitted financial aid apps table